### ISSUE TITLE
Apply trailing comma rule to function parameters

### ISSUE THREAD

**Issue by thasner** [palantir/tslint#1415]
### Bug Report

Typescript allows trailing comma in function parameter lists https://github.com/Microsoft/TypeScript/pull/8942

Could tslint apply the `trailing-comma` rule here, too?

- __TSLint version__: `3.13.0-dev.0`
- __TypeScript version__: `1.9.0-dev.20160531-1.0`
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
constructor(
        arg1: boolean,
        arg2: boolean,
        arg3: boolean
    ) {
```

with `tslint.json` configuration:

`tslint:latest`

#### Actual behavior

no error

#### Expected behavior

error: comma required at end of line 4

**Comment by thasner** [comment]
looks like this is a duplicate of #1409
